### PR TITLE
Privacy sandbox: intro/overview blog post draft

### LINKED ISSUE #1898: content: "Privacy Sandbox: Privacy Model for the Web"
Soft deadline: 3 Mar 2020
Author: @samdutton 
Reviewer: @kaycebasques

### PR DESCRIPTION
**Pull request by samdutton**
Fixes #1898.

### CODE REVIEW

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+The total number of people who see an ad.
+
+### Remarketing
+
+Advertising to people who've already visited your site. For example, an online store could show ads for a toy sale to people who previously viewed toys on their site.
+
+### Site
+
+(See above.)
```
Comment (kaycebasques) [inline src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md]: What is this supposed to be referring to?

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+### Surface
+
+(See Fingerprinting surface and also Passive surfaces above.)
```
Comment (kaycebasques) [inline src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md]: These should be links

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+
+### Policy and requirements
+
++   The [Tor Browser design's "Privacy Requirements"](https://2019.www.torproject.org/projects/torbrowser/design/#privacy), particularly the clear emphasis on "Unlinkability".
++   Mozilla anti-tracking policy: [wiki.mozilla.org/Security/Anti_tracking_policy](https://wiki.mozilla.org/Security/Anti_tracking_policy)
++   WebKit tracking prevention policy: [webkit.org/tracking-prevention-policy](https://webkit.org/tracking-prevention-policy/)
++   [Privacy Preserving Ad Click Attribution For the Web](https://webkit.org/blog/8943/privacy-preserving-ad-click-attribution-for-the-web/)
++   [Brave, Fingerprinting, and Privacy Budgets](https://brave.com/brave-fingerprinting-and-privacy-budgets/)
+
+## Appendix: Glossary of terms used in the explainers
```
[the remark this answers is not in the document]
Comment (samdutton) [reply src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md]: Good idea — you mean a separate post, right?

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md (hunk 1 above)
Comment (samdutton) [reply src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md]: I'll do a commit to fix this. It should refer to the eTLD and Domain entries.

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md (hunk 2 above)
Comment (samdutton) [reply src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md]: I'll do a commit to fix these.

**Comment by kaycebasques** [comment]
@samdutton just a reminder that you should be able to merge this when it's ready. I believe you're waiting on the go-ahead from others but I just want to re-iterate that I'm not going to merge this unless you give me an explicit approval.

**Comment by samdutton** [comment]
Thanks — yup, waiting for final review and go-ahead.

**Pull request merged**